packages/lib-utils/src/config.ts

```diff
@@ -10,6 +10,11 @@ export type UtilsConfig = {
    * with an appropriate error.
    */
   appFetch: ResourceFetch;
+
+  /**
+   * Kubernetes API server base URL (without the trailing slash).
+   */
+  k8sBaseURL: string;
```

Comment:
Interesting... I forgot this was jammed in at building of script time. I think we need to rethink this. 

I want to propose we actually do not do this in the SDK. I think this is on the App and thus should be something they prepend in their `appFetch` impl. Thus making our call in [commonFetch](https://github.com/openshift/dynamic-plugin-sdk/blob/6028af97c376111189edafcf59aea7a74d6881e0/packages/lib-utils/src/utils/common-fetch.ts#L14) being a protocol-less, host-less URL part. The host app can do:

```
const appFetch = (url: string, ...) => {
  //...
  fetch(`${baseURL}${url}`, ...);
  //...
}
```
This gives the power to the App to redirect as needed per-call. Some may go to KCP some can go to DevSandbox, etc etc etc... This gives more flexibility for us to configure out way through a multiple backend - one API situation.

Thoughts?

Reply:
I'm OK with this, but if we do this, we should rename `url` to `path` because it'd be just a part of the whole URL.

Reply:
Agreed, `path` rename as well please